tests/sles4sap/sap_deployment_automation_framework/execute_playbooks.pm

```diff
@@ -85,31 +87,25 @@ sub run {
     # Some playbooks use azure cli
     az_login();
 
-    # Register SUTs before executing playbooks
```

Comment:
Splitting block into a function `sdaf_register_byos` and moving rest after execution of playbook `pb_get-sshkey.yaml`